packages/api/src/external/commonwell/patient-demographics.ts

```diff
@@ -0,0 +1,95 @@
+import { Patient } from "@metriport/core/domain/patient";
+import { LinkDemographics } from "@metriport/core/domain/patient-demographics";
+import { PatientNetworkLink, GenderCodes } from "@metriport/commonwell-sdk";
+import {
+  scoreLink,
+  linkHasNewDemographics,
+  patientToNormalizedCoreDemographics,
+  removeInvalidArrayValues,
+  normalizeDob,
+  normalizeGender,
+  normalizeAndStringifyNames,
+  normalizeAddress,
+  stringifyAddress,
+  normalizeTelephone,
+  normalizeEmail,
+} from "../../domain/medical/patient-demographics";
+import { CwLink } from "./cw-patient-data";
+
+type CwGenderCode = `${GenderCodes}`;
+
+export function getNewDemographics(patient: Patient, links: CwLink[]): LinkDemographics[] {
+  const coreDemographics = patientToNormalizedCoreDemographics(patient);
+  const consolidatedLinkDemographics = patient.data.consolidatedLinkDemographics;
+  return getPatientNetworkLinks(links)
+    .map(patientNetworkLinkToNormalizedLinkDemographics)
+    .filter(linkDemographics => scoreLink({ coreDemographics, linkDemographics })[0])
+    .filter(
+      linkDemographics =>
+        linkHasNewDemographics({
+          coreDemographics,
+          consolidatedLinkDemographics,
+          linkDemographics,
+        })[0]
+    );
+}
+
+export function patientNetworkLinkToNormalizedLinkDemographics(
+  patientNetworkLink: PatientNetworkLink
+): LinkDemographics {
+  const dob = normalizeDob(patientNetworkLink.details.birthDate);
+  const cwGender = patientNetworkLink.details.gender.code as CwGenderCode;
+  const gender = normalizeGender(cwGender);
```

Comment:
`normalizeGender` gets a string, no need to cast to a specific type, right?

Reply:
Done -- I forget why I had that casting before but you're right, unnecessary.